test/views/PaymentHistoryViewSpec.scala

```diff
@@ -57,12 +60,18 @@ class PaymentHistoryViewSpec extends ViewSpec with ImplicitDateFormatter {
   }
 
   val testPayments: List[Payment] = List(
-    Payment(Some("AAAAA"), Some(10000), None, Some("Payment"), None, Some("lot"), Some("lotitem"), Some("2019-12-25"),
+    Payment(Some("AAAAA"), Some(10000), None, Some("Payment"), None, Some("lot"), Some("lotitem"), Some("2019-11-25"),
+      "2019-11-25", Some("DOCID01")),
+    Payment(Some("AAAAB"), Some(10000), None, Some("Payment"), None, Some("lot"), Some("lotitem"), Some("2019-12-25"),
       "2019-12-25", Some("DOCID01")),
-    Payment(Some("BBBBB"), Some(5000), None, Some("tnemyap"), None, Some("lot"), Some("lotitem"), Some("2007-03-23"),
-      "2019-12-25", Some("DOCID02"))
+    Payment(Some("BBBB1"), Some(5000), None, Some("tnemyap"), None, Some("lot"), Some("lotitem"), Some("2007-03-23"),
```

Comment:
We should either add some MFA credits in this list or adjust the paymentsMFA to check that the sorting is working for MFA

Reply:
* paymentsMFA test adjusted to verify sorting order is correct;